source/docs/magento1/search-engine.md

```diff
@@ -82,6 +90,146 @@ query Search {
 If you are using the default theme or the theme Chocolatine, the search bar
 should now be visible.
 
+<blockquote class="warning">
+⚠️ Common mistake: For magento 2 the elasticsuite does not index cms pages. To do that you need to install <a href="https://github.com/Smile-SA/magento2-module-elasticsuite-cms-search" target="_blank">Smile elasticsuite cms search module</a> on your magento instance. Failing to do so will cause the top search bar to malfunction. If you do not need the CMS Page search you can always override related `SearchBar` Components/GraphQL queries to disable the cms page search functionality as in the following diff:
+
+<details>
+<summary>
+Disable CMS Pages Search Diff
+</summary>
+
+#### Base theme:
+
+```
+diff --git a/src/web/theme/modules/Search/SearchBar/Autocomplete/Autocomplete.js b/src/web/theme/modules/Search/SearchBar/Autocomplete/Autocomplete.js
```

Comment:
this is a complete workaround to unlock the customer. IMHO, that should not be documented

Reply:
> unlock the customer

what do you mean this?

Reply:
this is in a toggleable item which means by default he cant see the diffs unless he clicks on them

Reply:
_unblock_ :)

even if it's toggleable, that's only a workaround either because the requirement is not properly documented or because there's a bug on our end. So let's fix the root cause and forget about it.